client/src/components/body/profilepage/Labs.js

```diff
@@ -1,47 +1,188 @@
-import React from "react";
+import React, { useEffect, useState } from "react";
 import LabGeneration from "../lab/LabGeneration";
 import useMainStateContext from "src/reducers/MainContext";
 import PropTypes from "prop-types";
+import { Input } from "reactstrap";
 
 const Labs = (props) => {
   const { actions, state } = useMainStateContext();
+
+  const [currentHeader, setCurrentHeader] = useState("");
+
+  const displayNotStartedLabs = () => {
+    setCurrentHeader("Not Started");
```

Comment:
Why do we have a "Not Started" filter? 

This is what the filter should be according to the figma: 
![Screenshot 2024-10-25 at 1 04 49 PM](https://github.com/user-attachments/assets/c042fe39-7508-48c4-b899-e3760ce2a138)